modules/tfm/tfm/boards/common/tfm_hal_platform.c

```diff
@@ -27,20 +31,22 @@ enum tfm_hal_status_t tfm_hal_platform_init(void)
 #if defined(TFM_PARTITION_CRYPTO)
 	int err;
 
+#if !defined(TFM_CRYPTO_RNG_MODULE_DISABLED)
```

Comment:
Let's remove this ! and flip the order to avoid the double-negative.

When I read a if(! foo),

I don't expect anything interesting to be happening when foo, because then it would have been written if(foo) else

Reply:
These are TF-Ms configuration and naming. So while I agree I don't want to change this.

Reply:
I'm not proposing to change the name of the config.

I'm proposing to remove the !

and then change the if-else order.

Reply:
Makes sense. Updated.